src/SonarScanner.MSBuild.Shim/ScannerEngineInput.cs

```diff
@@ -51,29 +45,27 @@ public ScannerEngineInput CloneWithoutSensitiveData()
         result.moduleKeys.UnionWith(moduleKeys);
         foreach (var property in scannerProperties)
         {
-            var key = property["key"].Value<string>();
-            var value = property["value"].Value<string>();
-            if (key == "sonar.modules")
-            {
-                result.modules.Value = value;
-            }
-            else
-            {
-                result.Add(key, ProcessRunnerArguments.ContainsSensitiveData(key) || ProcessRunnerArguments.ContainsSensitiveData(value) ? "***" : value);
-            }
+            var key = property.Key;
+            var value = property.Value;
```

Comment:
I don't think we need these anymore. They were previously difficult to compute and it made sense to precompute it. Now, it's not really needed and we can just use `property.Key` and `property.Value` everywhere
```suggestion
```